src/compile/scale/range.ts

```diff
@@ -221,7 +226,12 @@ export function defaultRange(
         // Only nominal data uses ordinal scale by default
         return type === 'nominal' ? 'category' : 'ordinal';
       } else if (isContinuousToDiscrete(scaleType)) {
-        return 'ordinal';
+        if (config.range && config.range.ordinal) {
```

Comment:
config.range is [provided by Vega](https://github.com/vega/vega-parser/blob/master/src/config.js#L201) so it will be falsy in VL code.

 Adding this clause mean the default won't ever be used.